<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Carte</class>
 <widget class="QMainWindow" name="Carte">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>631</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Carte</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="MapWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>520</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>21</width>
      <height>161</height>
     </rect>
    </property>
    <property name="maximum">
     <number>17</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>155</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>831</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="BouttonModifier">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BouttonAccesBDD">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>340</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Acces BDD</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>261</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BouttonCentrer">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Centrer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuBDD">
    <property name="title">
     <string>BDD</string>
    </property>
    <addaction name="actionChoix_BDD"/>
    <addaction name="actionGestion_BDD"/>
    <addaction name="actionExport_BDD"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuBDD"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionChoix_BDD">
   <property name="text">
    <string>Choix BDD</string>
   </property>
  </action>
  <action name="actionGestion_BDD">
   <property name="text">
    <string>Gestion BDD</string>
   </property>
  </action>
  <action name="actionExport_BDD">
   <property name="text">
    <string>Export BDD</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
